code/modules/games/cards.dm

```diff
@@ -0,0 +1,465 @@
+/datum/playingcard
```

Comment:
It'd be a good idea to add a proc to create and fill in the fields of a playing card, and then just do
`cards += new /datum/playingcard(foo, bar)`, to cut down on repeated lines